pyproject.toml

```
@@ -44,7 +45,7 @@ exclude = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-Django = ">=3.2,<5.1"
+Django = ">=3.2,<5.2"
```

Given 3.2 is no longer supported

```suggestion
Django = ">=4.2,<5.2"
```

Also, it would be good to drop Python 3.8 and add 3.12 to the matrix